schema/layers/precomposition-layer.json

```diff
@@ -40,6 +40,11 @@
                     "title": "Start Time",
                     "type": "number",
                     "default": 0
+                },
+                "tm": {
+                    "title": "Time Remap",
+                    "description": "Timeline remap function",
```

Comment:
Nit: Mention that the values are expressed in seconds.

Reply:
Done.